### DISCUSSION TITLE
Complement function-补数功能

### DISCUSSION THREAD

**Discussion by Laughingjun** [apache/dolphinscheduler#7125] [Q&A] [answered]
Select a time range of a month in the previous year and perform data repleniting. How can the time in each SQL statement be automatically changed to the selected time

**Comment by zhongjiajie** [comment, marked as answer]
Hi @Laughingjun , to discuss or share your idea in community, please use English first for more general, thanks

**Reply by zhongjiajie** [reply]
@Laughingjun  Thanks for changing the title, you could set your SQL task time build-in parameter, which it would calculate by the day task run. Wish it would help you.
